docs/semgrep-assistant/overview.md

```diff
@@ -77,6 +77,15 @@ You can view findings for which Assistant has generated autofix snippets in Semg
 
 The finding's detail page also includes a link to the PR or MR comment with the autofix; you can go to the PR or MR, where you can directly commit the autofix. <!-- #10516 --> <!-- This isn't unique to Assistant; it's available for autofix and PR/MR comments in general. -->
 
+### Priority inbox
+
+Semgrep sends weekly emails with information on the top three backlog tasks according to Assistant. These emails are sent out on Monday to all organization admins.
```

Comment:
Is this for all products purchased by or enabled for the user?

Reply:
Yes Priority Inbox looks at all 3 products and will provide results if the end user has all 3.

Reply:
Is it worth indicating that this is across products?

Reply:
Added a mention that this is across all products!